src/sentinel1_reader/sentinel1_burst_slc.py

```diff
@@ -218,7 +220,31 @@ def as_isce3_radargrid(self):
                                                  width,
                                                  ref_epoch)
 
-    def to_vrt_file(self, out_path):
+    def slc_to_gtiff(self, out_path):
+        '''Write burst to GTiff file.
+
+        Parameters:
+        -----------
+        out_path : string
+            Path of output GTiff file.
+        '''
+        # get output directory of out_path
+        dst_dir, _ = os.path.split(out_path)
+
+        # create temporary VRT
+        temp_vrt = tempfile.NamedTemporaryFile(dir=dst_dir, suffix='.tif')
+        self.slc_to_vrt_file(temp_vrt.name)
+
+        # open temporary VRT and translate to GTiff
+        src_ds = gdal.Open(temp_vrt.name)
+        dst_ds = gdal.Translate(out_path, src_ds, format='GTiff')
+
+        # clean up
+        src_ds = None
+        dst_ds = None
```

Comment:
I don't use gdal very frequently, just curious that regarding the codacy flag, can we change these lines to below?

```python
gdal.Translate(out_path, src_ds, format='GTiff')

# clean up
src_ds = None
```